SpatialGDK/Source/SpatialGDK/Private/SpatialView/OpList/EntityComponentOpList.cpp

```diff
@@ -57,9 +77,22 @@ EntityComponentOpListBuilder& EntityComponentOpListBuilder::SetAuthority(Worker_
 	return *this;
 }
 
+EntityComponentOpListBuilder& EntityComponentOpListBuilder::SetDisconnect(Worker_ConnectionStatusCode StatusCode, FString DisconnectReason)
+{
+	// Convert an FString to a char* that we can store.
+	FTCHARToUTF8 Reason(*DisconnectReason);
```

Comment:
Discussed offline - this string needs to be owned somewhere in the oplist. Probably a TArray<FTCHARToUTF8> in the oplist.